python/pyspark/sql/functions.py

```diff
@@ -3471,7 +3473,7 @@ def from_json(col, schema, options={}):
     return Column(jc)
 
 
-def to_json(col, options={}):
+def to_json(col, options=None):
```

Comment:
Seems like we still have to modify a few annotations, right?

Probably something like [functions.pyi.patch.txt](https://github.com/apache/spark/files/5583207/functions.pyi.patch.txt)

Reply:
Good catch, I've added them

Reply:
Not sure why implicit optional is still allowed 🤔